solc/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "solc"
```

Comment:
Is this going to be a wrapper around solc, or are we planning to port solc to Rust?